### PR TITLE
Make error messages understandable

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by bratseth** [comment]
@gjoranv please review

File: config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java [hunk 1]
```diff
@@ -479,7 +480,8 @@ public void setDocumentApi(ContainerDocumentApi containerDocumentApi) {
     @NonNull
     public DocprocChains getDocprocChains() {
         if (containerDocproc == null)
-            throw new IllegalStateException("Null docproc components!");
+            throw new IllegalStateException("Search components not found in container cluster '" + getSubId() +
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java]: Copy/paste error. I think you mean 'document-processing'.

File: config-model/src/test/java/com/yahoo/config/model/provision/ModelProvisioningTest.java [hunk 2]
```diff
@@ -1356,6 +1356,51 @@ public void testMultitenantButNotHosted() {
         assertEquals(1, controller.getContainers().size());
     }
 
+    @Test
+    public void testModelWithReferencedIndexingCluster() {
+        String services =
+                "<?xml version=\"1.0\" encoding=\"utf-8\" ?>\n" +
```
Comment (gjoranv) [inline config-model/src/test/java/com/yahoo/config/model/provision/ModelProvisioningTest.java]: Again, you could use a helper to avoid the newlines.

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java]: Fixed - thanks!

**Comment by gjoranv** [comment]
👍

**Pull request merged**